pages/Topic.jsx

```diff
@@ -0,0 +1,13 @@
+import { useParams } from "react-router"
+import ArticleList from "../components/ArticleList"
+
+export default function Topic () {
+
+    let { topic } = useParams()
```

Comment:
Might have been easier just to get the topic in two places, rather than get it here then pass it through two components - but up to you.